Assets/Scripts/SafeZoneLogic.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SafeZoneLogic : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject player;
+    [SerializeField]
+    private int radius;
+    private GameObject[] pickables;
+    private int returnedPickables;
+
+    void Start()
+    {
+        pickables = GameObject.FindGameObjectsWithTag("PickableObject");
+    }
+
+    void Update()
+    {
+        if (player.GetComponent<HoldObject>().get_PickedUpObject() == null)
+        {
+            if (IsTargetInRange(player, radius) == true)
+            {
+                //Add extinguish ammo
+            }
+        }
+
+        returnedPickables = 0;
+        for (int i = 0; i < pickables.Length; i++)
+        {
+            if (IsTargetInRange(pickables[i], radius) == true)
+            {
+                returnedPickables++;
+            }
```

Comment:
```suggestion
            if (IsTargetInRange(pickables[i], radius))
            {
                returnedPickables++;
            }
```
Disregard if there's a specific reason behind using `== true` here